### PR TITLE
Fix .rubocop_todo.yml

### PR DESCRIPTION
**Pull request by pirj**
Take a shortcut compared to #982

My apologies it took me nearly two years :smile:

---

Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [-] Added tests.
* [-] Updated documentation.
* [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-# Note that changes in the inspected code, or installation of new
-# versions of RuboCop, may require this file to be generated again.
-
-# Offense count: 1
-# Configuration parameters: CountComments, CountAsOne.
-Metrics/ClassLength:
-  Max: 106
-
-RSpec/ExampleLength:
-  Exclude:
```
Comment (pirj) [inline .rubocop_todo.yml:15]: Doesn't seem to cause any trouble anymore

File: lib/rubocop/cop/rspec/described_class.rb [hunk 2]
```diff
@@ -129,17 +128,13 @@ def message(offense)
         end
 
         def scope_change?(node)
-          scope_changing_syntax?(node)          ||
```
Comment (pirj) [inline lib/rubocop/cop/rspec/described_class.rb:132]: Styling

File: lib/rubocop/cop/rspec/described_class.rb [hunk 3]
```diff
@@ -152,6 +147,7 @@ def offensive?(node)
 
         def offensive_described_class?(node)
           return unless node.const_type?
+
```
Comment (pirj) [inline lib/rubocop/cop/rspec/described_class.rb:150]: Styling. I found it hard to understand what guard statement this comment relates to.

File: lib/rubocop/cop/rspec/described_class.rb [hunk 4]
```diff
@@ -196,9 +191,7 @@ def collapse_namespace(namespace, const)
         #   const_name(s(:const, s(:const, nil, :M), :C)) # => [:M, :C]
         #   const_name(s(:const, s(:cbase), :C))          # => [nil, :C]
         def const_name(node)
-          # rubocop:disable InternalAffairs/NodeDestructuring
-          namespace, name = *node
-          # rubocop:enable InternalAffairs/NodeDestructuring
+          namespace, name = *node # rubocop:disable InternalAffairs/NodeDestructuring
```
Comment (pirj) [inline lib/rubocop/cop/rspec/described_class.rb:194]: Styling.

File: lib/rubocop/cop/rspec/described_class.rb [hunk 5]
```diff
@@ -196,16 +191,13 @@ def collapse_namespace(namespace, const)
...
           if !namespace
             [name]
           elsif namespace.const_type?
             [*const_name(namespace), name]
           elsif %i[lvar cbase send].include?(namespace.type)
             [nil, name]
-
```
Comment (bquorning) [inline lib/rubocop/cop/rspec/described_class.rb]: Feel free to squash these blank-line removals into the previous commit. I have no idea why I added them there 😬

**Review by bquorning — APPROVED** [review]
I had _completely_ forgotten about #982. I still think it makes sense to extract the logic instead of just “inline methods and remove line breaks” as this PR seems to do. Or to follow @Darhazer’s advice and look towards rubocop-ast (and perhaps move some logic from our repository to their repository).

**Review by Darhazer — APPROVED** [review]

**Pull request merged**